libcentrifugo/engine/engineredis/hash.go

```diff
@@ -0,0 +1,26 @@
+package engineredis
+
+import (
+	"hash/fnv"
+)
+
+// Adapted from from https://github.com/dgryski/go-jump package by Damian Gryski.
+// Hash consistently chooses a hash bucket number in the range [0, numBuckets) for the given key.
+// numBuckets must be >= 1.
+func Hash(ch string, numBuckets int) int32 {
+
+	hash := fnv.New64a()
+	hash.Write([]byte(ch))
+	key := hash.Sum64()
+
+	var b int64 = -1
+	var j int64
+
+	for j < int64(numBuckets) {
+		b = j
+		key = key*2862933555777941757 + 1
```

Comment:
I assume go does the right thing and interprets that literal as an int64 here especially given the multiplication with an int64 but did you verify that? truncating that number accidentally would destroy the properties of this hash.